### PR TITLE
support getFsInfo()

### PR DESCRIPTION
**Pull request by heartlock**
Fixes: #110

Signed-off-by: heartlock 21521209@zju.edu.cn

### CODE REVIEW

**Comment by feiskyer** [comment]
@heartlock Could you add cadvisor releated code to https://github.com/hyperhq/cadvisor first and update cadvisor vendor here?

**Comment by heartlock** [comment]
file:///home/chen/Pictures/Screenshot%20from%202016-08-21%2023-20-40.jpg

**Comment by heartlock** [comment]
sure,I will do it @feiskyer

**Comment by resouer** [comment]
@heartlock Update vendor of hypernetes is a little bit tricky.
1. commit your changes to `hyperhq/cadvisor`
2. update all cadvisor lines in `hypernetes/Godeps.json` to new commitID of `hyperhq/cadvisor`
3. copy all contents in`hyperhq/cadvisor` except `/docs`, `*_test.go` into `vendor/src/github.com/google/cadvisor`
4. test

**Comment by heartlock** [comment]
print fsinfo
image

**Comment by resouer** [comment]
@heatlock why Fail to check disk when you have already supported `getFsInfo`?

File: pkg/kubelet/disk_manager.go [hunk 1]
```diff
@@ -93,6 +93,7 @@ func (dm *realDiskSpaceManager) IsRootDiskSpaceAvailable() (bool, error) {
 
 func (dm *realDiskSpaceManager) isSpaceAvailable(fsType string, threshold int, f func() (cadvisorapi.FsInfo, error)) (bool, error) {
 	fsInfo, err := dm.getFsInfo(fsType, f)
+
```
Comment (feiskyer) [inline pkg/kubelet/disk_manager.go]: remove empty line here.

**Comment by resouer** [comment]
@heartlock Please remove `*.md`,`.gitignore`, `.git`, `*.txt` from vendor.

And I see cadvisor in hypernetes/Godeps.json is be6e26b8db65c92b8ca346920213c19ea1332f5e, but the latest commit in hyperhq/cadvisor is 30a83424c239fc613953b3600b53c843ea47a586

Others lgtm

**Pull request merged**